man/git-bulk.md

```diff
@@ -0,0 +1,103 @@
+git-bulk(1) -- Run git commands on multiple repositories
+========================================================
+
+## SYNOPSIS
+
+usage: usage: git bulk [-g] ([-a]|[-w &lt;ws-name&gt;]) &lt;git command&gt;
+       git bulk --addworkspace &lt;ws-name&gt; &lt;ws-root-directory&gt;
+       git bulk --removeworkspace &lt;ws-name&gt;
+       git bulk --addcurrent &lt;ws-name&gt;
+       git bulk --purge
+       git bulk --listall
+
+## DESCRIPTION
+
+git bulk adds covenien support for operations that you want to execute on multiple git repositories.
```

Comment:
And don't forget to rebuild git-bulk.1 and git-bulk.html, after correcting the typo.